bchangg/on-air-device
# Feature 41/connect rails app to hivemq

# Description, Motivation, and Context

Initial proof of concept to connect to the MQTT broker.
This endpoint will be exposed to the react app to allow the user to turn a light on or off eventually.

# Replication Steps / How to test

1. Log in to HiveMQ with the correct credentials
2. Subscribe your HiveMQ client to the topic `light`
3. Sent a `post` request to `api/v1/mqtt/publish` 
4. Observe that a new message `on` pops up in the web client view
5. Observe that the local server returns a 200 success json response

~NOTE: Working on adding some tests atm~
